### PR TITLE
CI check for every commits

### PR DESCRIPTION
**Pull request by mariotaku**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install system dependencies
+      run: npm install -g @webosose/ares-cli
```
Comment (Informatic) [inline .github/workflows/main.yml:29]: This is very likely not required - we already have a pinned dependency on `@webosose/ares-cli` in our `package.json`, and it should be accessible in `npm` scripts used below (or directly via `node_modules/.bin/ares-...`)

**Comment by Informatic** [comment]
Fail seems to be caused by missing git submodules.

File: .github/workflows/main.yml (hunk 1 above)
Comment (mariotaku) [reply .github/workflows/main.yml:33]: Ah I see! Correcting that.

**Comment by mariotaku** [comment]
See #6

**Comment by mariotaku** [comment]
I'll treat each tag as a release and trigger builds when new tags pushed. Like moonlight-tv. What do you think?

**Comment by Informatic** [comment]
Sounds good to me.

**Comment by mariotaku** [comment]
Tested: https://github.com/mariotaku/webos-homebrew-channel/actions/runs/778013920

**Comment by mariotaku** [comment]
It's ready to be merged :)

**Comment by Informatic** [comment]
LGTM, thanks.

**Pull request merged**